tracer/src/Datadog.Trace.Tools.Runner/Checks/Resources.cs

```diff
@@ -49,6 +52,12 @@ internal static class Resources
 
         public static string SuspiciousRegistryKey(string key) => $@"The registry key HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\.NETFramework\{key} is defined and could prevent the tracer from working properly. Please check that all external profilers have been uninstalled properly.";
 
+        public static string MissingRegistryKey(string key) => $@"The registry key {key} is missing. Make sure the tracer has been properly installed with the MSI.";
+
+        public static string MissingProfilerRegistry(string path) => $@"The profiler was installed to the path {path} but the file is missing or you don't have sufficient permission. Try reinstalling the tracer with the MSI and check the permissions.";
```

Comment:
I'd suggest writing a message similar to `WrongProfilerRegistry`:

```suggestion
        public static string MissingProfilerRegistry(string registryKey, string path) => $@"The registry key {registryKey} was set to '{path}' but the file is missing or you don't have sufficient permission. Try reinstalling the tracer with the MSI and check the permissions.";
```